src/TrackRowInfoControl.js

```diff
@@ -75,28 +83,32 @@ export default function TrackRowInfoControl(props){
         buttons.push({
             onClick: onSortAscClick,
             icon: SORT_ASC,
-            title: "Sort rows in ascending order"
+            title: "Sort rows in ascending order",
+            highlit: sortAsceButtonHighlit
         });
         buttons.push({
             onClick: onSortDescClick,
             icon: SORT_DESC,
-            title: "Sort rows in descending order"
+            title: "Sort rows in descending order",
+            highlit: sortDescButtonHighlit
         });
     }
 
     if(onSearchRows) {
         buttons.push({
             onClick: onSearchClick,
             icon: FILTER,
-            title: "Search keywords"
+            title: "Filter rows",
+            highlit: filterButtonHighlit
         });
     }
 
     if(onFilterRows) {
         buttons.push({
             onClick: onReset,
             icon: RESET,
-            title: "Remove all filters"
+            title: "Remove all filters",
+            isHighlit: false
```

Comment:
Is this a typo, or does `highlit` mean something different than `isHighlit`?

Reply:
That was a typo, sorry for the confusion.